patterns/2-structured/maturity-model.md

```diff
@@ -4,86 +4,69 @@ Maturity Model
 
-* PP-0: Individuals and teams do not regularly disclose their plans or products to multiple stakeholders. No formal actions exists at the organization.
+* PP-0: Individuals and teams do not regularly disclose their plans or products to multiple stakeholders. No formal actions exist in the organization.
 * PP-1: Individuals and teams give visibility to their plans or products to multiple stakeholders, before they are started. Shared roadmap.
 * PP-2: There are already shared roadmaps with clear guidelines and contribution rules where stakeholders can provide feedback. However, this is not standardized across the organization and not all of the projects provide this info.
-* PP-3: Roadmaps are shared by default and there is a standard process and homogeneous way to do this across the organization at the level of each InnerSource project. This contains clear rules to contribute and influence in the roadmap.
+* PP-3: Roadmaps are shared by default and there is a standard process and homogeneous way to do this across the organization at the level of each InnerSource project. This contains clear rules to contribute to and influence the roadmap.
 
-**Development Process & Tools**
+#### Development Process & Tools
 
-InnerSource projects thrive when contributors become active and participate. As a result, making contribution easier should be balanced with pure technical goals.
+InnerSource projects thrive when contributors become active and participate. As a result, making contributions easier should be balanced with pure technical goals.
 
-* DP-0: Each team follows its own development process and tools. They are not defined to share knowledge and artifacts outside development team. Siloed development teams.
-* DP-1: Development teams use shared code repositories, internally. Some teams develop their own CI process, using non corporate or standard CI tools. There is no code review process defined, although some projects teams do it internally.
-* DP-2: The organization sponsors and promotes a shared repository for collective knowledge. Some teams develop their own CI process, using corporate CI tools. There are CI environments. Code review process defined, and used by some projects. Sometimes code review is done by outside team members.
+* DP-0: Each team follows its development process and tools. They are not defined to share knowledge and artifacts outside the development team. Siloed development teams.
+* DP-1: Development teams use shared code repositories, internally. Some teams develop their own CI process, using non-corporate or standard CI tools. There is no code review process defined, although some project teams do it internally.
+* DP-2: The organization sponsors and promotes a shared repository for collective knowledge. Some teams develop their own CI process, using corporate CI tools. There are CI environments. The code review process is defined, and used by some projects. Sometimes code review is done by outside team members.
```

Comment:
```suggestion
* DP-2: The organization sponsors and promotes a shared repository for collective knowledge. Some teams develop their own CI process, using corporate CI tools. There are CI environments. A code review process is defined, and used by some projects. Sometimes code review is done by outside team members.
```

Not sure about the/a here.
Whatever we decide here, we should use the same phrase in DP-3.